OpenProblemLibrary/Hope/Multi1/03-05-Basis-subspace/Basis_30.pg

```diff
@@ -102,13 +138,13 @@ c. If possible, write [` \mathbf{a_4} `] as a linear combination of [` \mathbf{a
 
 d. The dimension of the column space of [` A `] is [_____]{"2"}, and the column space of [` A `] is a subspace of [@ ANS($w1->cmp); $w1->ans_rule(5); @]*.
 
-e. Find a basis for the column space of [` A `].  Enter your answer as a comma separated list of vectors.  Each vector should have the form [| <a,b,c> |] or [| <a,b,c,d> |] where a,b,... are numbers.  Do not use the symbols [` \mathbf{a_1, a_2, \ldots} `] in your answers. 
- 
-    A basis for the column space of [`A`] is [`\big\lbrace`] [_________________________]{List(Vector(1,0,0),Vector(0,1,0))->cmp(list_checker=>~~&basis_checker_list_of_vectors)} [`\big\rbrace`]
+e. Find a basis for the column space of of [` A `].  Enter your answer as a comma separated list of vectors.  Each vector should have the form [| <a,b,c> |] or [| <a,b,c,d> |] where a,b,... are numbers.  Do not use the symbols [` \mathbf{a_1, a_2, \ldots} `] in your answers.   
```

Comment:
It looks like this is introducing a typo ("of of").  Also, the instructions need to change since the correct answer requires students to enter symbols.  It's not necessary to tell students not to enter explicit vectors since the checker handles them (though they will always be marked wrong), unless this is intended as a hint that the correct answer will be in terms of a1, a2, a3, a4.

Reply:
Oh no. My bad, I somehow cut-and-pasted an earlier version of the question back. Yes, you are right, it's not necessary to tell students not to enter explicit vectors.